### ISSUE TITLE
Ping exact FCS version

### ISSUE THREAD

**Issue by nojaf** [fsprojects/fantomas#2118]
In the latest release we claim that we need FCS 41.0.1 or higher:
![image](https://user-images.githubusercontent.com/2621499/156004670-bc4af187-5678-4d37-aa6d-98f914c8397a.png)

This is not the case though, Fantomas will only support one exact FCS version (being `41.0.1` in this case).

Do you agree with pinning this version @baronfel?

**Comment by baronfel** [comment]
Yeah, though we should push for greater reliability on the versions of FCS published from dotnet/fsharp to help prevent these kinds of mixups.

**Comment by nojaf** [comment]
Interestingly enough, we seem to already specify an exact version in the `paket.dependencies` file:
https://github.com/fsprojects/fantomas/blob/17e6b451c74de7e60a86f59bbc3d209a565dbde4/paket.dependencies#L7

**Comment by nojaf** [comment]
Another somewhat related thing:
![image](https://user-images.githubusercontent.com/2621499/156331130-294bc4a2-db68-4c6a-964a-e39c48dcd62a.png)

Fantomas.Extras should also depend on the exact same version of Fantomas.
This constraint is also wrong.
